ivy/functional/backends/paddle/experimental/statistical.py

```diff
@@ -537,8 +514,9 @@ def cov(
     )
 
 
-@with_unsupported_device_and_dtypes(
-    {"2.5.1 and below": {"cpu": ("uint16", "bfloat16")}}, backend_version
+@with_supported_dtypes(
+    {"2.5.1 and below": ("complex", "bool", "float32", "float64")},
+    backend_version,
 )
 def cummax(
```

Comment:
same problem as `nanmean `and `nanmedian`

Reply:
same solution